### PR TITLE
docs(build): add more example

### PR DESCRIPTION
**Pull request by tpoisonooo**
## Motivation

add more build example for deploee.

## Modification

update doc.

### CODE REVIEW

**Review by lvhan028 — APPROVED** [review]
LGTM

File: docs/zh_cn/01-how-to-build/linux-x86_64.md [hunk 1]
```diff
@@ -390,3 +390,30 @@ mim install -e .
...
+      -DMMDEPLOY_BUILD_SDK=ON \
+      -DMMDEPLOY_BUILD_EXAMPLES=ON \
+      -DMMDEPLOY_BUILD_SDK_PYTHON_API=ON \
+      -DMMDEPLOY_TARGET_DEVICES="cuda;cpu" \
+      -DMMDEPLOY_TARGET_BACKENDS="trt;ort;ncnn;openvino" \
+      -Dpplcv_DIR=${PPLCV_DIR}/cuda-build/install/lib/cmake/ppl \
+      -DTENSORRT_DIR=${TENSORRT_DIR} \
+      -DCUDNN_DIR=${CUDNN_DIR} \
+      -DONNXRUNTIME_DIR=${ONNXRUNTIME_DIR} \
+      -DInferenceEngine_DIR=${InferenceEngine_DIR} \
```
Comment (AllentDan) [inline docs/zh_cn/01-how-to-build/linux-x86_64.md:411]: `InferenceEngine_DIR` 好像没给过定义

Comment (tpoisonooo) [reply docs/zh_cn/01-how-to-build/linux-x86_64.md:411]: > 在 CMakeLists 里 https://github.com/search?q=repo%3Aopen-mmlab%2Fmmdeploy%20InferenceEngine_DIR&type=code

**Review by AllentDan — APPROVED** [review]
LGTM

**Comment by AllentDan** [comment]
Lint failed.

**Pull request merged**